ledger/participant-integration-api/src/main/scala/platform/store/backend/common/CommonStorageBackend.scala

```diff
@@ -538,8 +539,42 @@ private[backend] trait CommonStorageBackend[DB_BATCH] extends StorageBackend[DB_
           where
             delete_events.event_offset <= $pruneUpToInclusive"""
     }(connection, loggingContext)
+
+    if (pruneAllDivulgedContracts) {
```

Comment:
Consider moving this to just after the `create events pruning` to increase cache locality on the `participant_create_events` table.

Reply:
Done